arch/arm/boot/compressed/Makefile

```diff
@@ -148,7 +152,7 @@ LDFLAGS_vmlinux += -X
 LDFLAGS_vmlinux += -T
 
 # For __aeabi_uidivmod
-lib1funcs = $(obj)/lib1funcs.o
+lib1funcs = $(obj)/lib1funcs.o $(obj)/divdi3.o
```

Comment:
Looks like we don't really need it on any platform we currently have. Will it break? whoooooooo knows :)